src/main/java/dev/dirs/Util.java

```diff
@@ -183,27 +194,30 @@ private static void initBase64Encoding() {
     }
   }
 
-  private static String[] runCommands(int expectedResultLines, Charset charset, String... commands) {
-    final ProcessBuilder processBuilder = new ProcessBuilder(commands);
+  private static String[] runCommands(int expectedResultLines, Charset charset, String... commands) throws IOException {
+    final String[] fullCommand = new String[commands.length + 2];
+    fullCommand[0] = "cmd.exe";
+    fullCommand[1] = "/C";
+    for (int i = 0; i < commands.length; ++i) fullCommand[i + 2] = commands[i];
+    final ProcessBuilder processBuilder = new ProcessBuilder(fullCommand);
+    final Map<String, String> env = processBuilder.environment();
+    final String path = env.get("Path");
+    final String toAppend = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files\\PowerShell\\7";
+    env.put("Path", (path == null ? "" : path) + toAppend);
```

Comment:
I tested on a windows vm that if I removed the Path field entirely, I reproduced the error in #29. If I set `Path` just toAppend (i.e. removed the jvm Path entirely), things worked as expected for both pwsh.exe and powershell.exe.